modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java

```diff
@@ -1168,6 +1168,34 @@ public void testAddObjectEntryAsDraft() throws Exception {
 		}
 	}
 
+	@Test
+	public void testAddObjectEntryToStagingGroup() throws Exception {
+		Group liveGroup = GroupTestUtil.addGroup();
+
+		GroupTestUtil.enableLocalStaging(
+			liveGroup, TestPropsValues.getUserId());
+
+		Group stagingGroup = liveGroup.getStagingGroup();
+
+		_objectDefinition.setScope(ObjectDefinitionConstants.SCOPE_SITE);
```

Comment:
@peerkar would you mind creating a new custom object like [here](https://github.com/liferay/liferay-portal/blob/master/modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectEntryLocalServiceTest.java#L3193-L3200)